data/rulesets.ts

```diff
@@ -1510,4 +1510,51 @@ export const Rulesets: {[k: string]: FormatData} = {
 			pokemon.trapped = true;
 		},
 	},
+	chimera1v1rule: {
+		effectType: 'Rule',
+		name: 'Chimera 1v1 Rule',
+		desc: "Validation and battle effects for Chimera 1v1.",
+		ruleset: ['Team Preview', 'Max Team Size = 6', 'Min Team Size = 6', 'Picked Team Size = 6'],
```

Comment:
```suggestion
		ruleset: ['Team Preview', 'Picked Team Size = 6'],
```

Max Team Size 6 is the default and Min Team Size defaults to Picked Team Size. Neither of these should be defined.